aspnetcore/toc.md

```diff
@@ -160,6 +160,7 @@
 ## [Filter methods for Razor Pages](xref:razor-pages/filter)
 ## [Create a Razor Class Library](xref:razor-pages/ui-class)
 ## [Route and app conventions](xref:razor-pages/razor-pages-conventions)
+## [Partial views](xref:mvc/views/partial)
```

Comment:
I recommend removing this ToC link. By putting it here, the Razor Pages ToC node appears for versions prior to 2.0:
![image](https://user-images.githubusercontent.com/10702007/45438132-fc01a900-b67b-11e8-9d07-c5a3e79bda39.png)